compass/landice/tests/humboldt/__init__.py

```diff
@@ -14,4 +17,61 @@ def __init__(self, mpas_core):
         super().__init__(mpas_core=mpas_core, name='humboldt')
 
         self.add_test_case(
-            Default(test_group=self))
+            MeshGen(test_group=self))
+
+        # Set up tests without calving using the 1km mesh
+        mesh_type = '1km'
+        for velo_solver in ['FO', ]:
```

Comment:
@trhille , I was trying to keep the number of new tests added by this PR to a minimum, so I deliberately only included the FO solver for the 1km mesh, since none of the 1km tests are in any suites.  My thought was it would be nice to be able to set up a high res Humboldt test with COMPASS in a standardized way.  I'm open to adding other versions of this test if you think they'd be useful.